apps/wallet/src/ui/app/shared/faucet/request-button/index.tsx

```diff
@@ -16,40 +18,90 @@ import type { ButtonProps } from '_app/shared/button';
 
 import st from './RequestButton.module.scss';
 
+//TODO:  Remove post testnet
+const TESTNET_DISCORD_LINK =
+    'https://discord.com/channels/916379725201563759/1037811694564560966';
+
 type FaucetRequestButtonProps = {
     mode?: ButtonProps['mode'];
     trackEventSource: 'home' | 'settings';
 };
 
+function FaucetDiscordLink({
```

Comment:
To avoid the extra function and duplication just move it in the other component

Reply:
done